crates/pixi_manifest/Cargo.toml

```diff
@@ -40,14 +40,14 @@ rattler_virtual_packages = { workspace = true }
 
 # TODO: Remove these dependencies
 console = { workspace = true }
-glob = "0.3.1"
 miette = { workspace = true }
 
 [dev-dependencies]
 assert_matches = { workspace = true }
 fancy_display = { workspace = true }
+glob = "0.3.1"
 insta = { workspace = true, features = ["yaml"] }
-pixi_consts = { workspace = true }
+pixi_config = { workspace = true }
```

Comment:
Id rather not add this dependency if its not needed. I would prefer a little duplication.

Reply:
Reverted